service/src/main/resources/application.yml

```diff
@@ -10,6 +10,9 @@ logging:
         cloud:
           gateway: ERROR
 
+zuul:
```

Comment:
Does this do anything? We're using spring cloud gateway and not zuul.

Reply:
Sorry, removed. Leftover of previous experiments.